lib/di.dart

```diff
@@ -276,6 +276,7 @@ Future<void> setup({
 
   if (!_isSetupFinished) {
     getIt.registerSingletonAsync<SharedPreferences>(() => SharedPreferences.getInstance());
+    getIt.registerSingleton<FlutterSecureStorage>(secureStorage);
```

Comment:
nothing to be changed, but was there a reason for this change? or even for those `if (!_isSetupFinished)` conditions?

Reply:
it's so that we have an instance of secureStorage when loading the settingsStore
I'm not totally sure what the _isSetupFinished is all about, I just assumed it should follow the same pattern that sharedpreferences does and that it'd fit here